.github/workflows/ci.yml

```diff
@@ -45,7 +45,7 @@ jobs:
             dwolla/jenkins-agent-core
           # Docker tags based on the following events/attributes
           tags: |
-            type=sha,suffix=-${{ matrix.temurin_tag }}
+            prefix=${{ matrix.temurin_tag }}-,type=sha
```

Comment:
This might be a little more clear written as `type=sha,prefix=${{ matrix.temurin_tag }}-` but we're pretty deep into bikeshed territory now, so YMMV